nemo_rl/algorithms/sft.py

```diff
@@ -325,25 +364,23 @@ def validate(
                 )
             else:
                 if model_type == "lm":
-                    list_of_val_metrics.append(
-                        SFTValMetrics(val_loss=float(val_results["loss"]))
-                    )
+                    dict_val_metrics["val_loss"].append(float(val_results["loss"]))
```

Comment:
Not directly related to your changes but can you explain why the SFT path uses directly `val_results["loss"]` while the RM path needs to go through `val_results["all_mb_metrics"]["loss"]`? This seems a bit inconsistent.

Reply:
Sounds good, resolving!